src/caf/toolkit/cost_utils.py

```diff
@@ -78,6 +82,10 @@ def check_df_col_names(self) -> CostDistribution:
         for col_name, col_val in cols.items():
             if col_val not in self.df:
                 err_cols.update({col_name: col_val})
+        cols.update({"weighted_avg_col": getattr(self, "weighted_avg_col")})
```

Comment:
Why are you doing this here instead of on line 78 where the rest of the columns are defined?

Reply:
So it isn't included in the check for if the columns are in the df. This means that for input dists they won't error in the old format..